src/main/java/org/minions/devfund/marcos/Squarelotron.java

```diff
@@ -0,0 +1,193 @@
+package org.minions.devfund.marcos;
+
+import java.util.Arrays;
+import java.util.stream.IntStream;
+
+/**
+ * It is in charge to represent a square matrix and it is operations.
+ */
+public class Squarelotron {
+
+    private int[][] squarelotron;
+    private int size;
+
+    /**
+     * Initializes an instance of {@link Squarelotron}.
+     *
+     * @param n square matrix dimension.
+     */
+    public Squarelotron(final int n) {
+        this.size = n;
+        squarelotron = new int[n][n];
+        fillMatrix();
+    }
+
+    /**
+     * Gets the Squarelotron matrix.
+     *
+     * @return square matrix.
+     */
+    public int[][] getSquarelotron() {
+        return Arrays.copyOf(squarelotron, size);
+    }
+
+    /**
+     * Performs the Up Side Down Flip operation.
+     *
+     * @param ring matrix ring number.
+     * @return {@link Squarelotron}.
+     */
+    public Squarelotron upsideDownFlip(final int ring) {
+        Squarelotron squarelotron = new Squarelotron(this.size);

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "squarelotron" which hides the field declared at line 11. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)